### PR TITLE
Remove racist language

### PR DESCRIPTION
**Pull request by fulldecent**
This removes deprecated language

### CODE REVIEW

**Comment by MicahZoltu** [comment]
This is a non-normative change to a final PR, so needs both editor approval and author approval.
cc William Entriken <github.com@phor.net>, Dieter Shirley <dete@axiomzen.co>, Jacob Evans <jacob@dekz.net>, Nastassia Sachs <nastassia.sachs@protonmail.com> (sadly, none of which have GitHub handles).

**Comment by fulldecent** [comment]
Approved.

Thank you @MicahZoltu

**Comment by axic** [comment]
@fulldecent do you want to take the opportunity to change your email to a GitHub username so that any future changes can be easily identified?

**Comment by fulldecent** [comment]
@axic Thank you. Updated. 

I'm very happy to have LESS work for you.

**Review by MicahZoltu — APPROVED** [review]

**Comment by Zachinquarantine** [comment]
Please explain how that's racist.....

**Comment by fulldecent** [comment]
Explained here https://fulldecent.blogspot.com/2021/07/whitelist-and-blacklist.html

**Comment by Zachinquarantine** [comment]
> But those have been deprecated

Says who?

**Comment by Garito** [comment]
Is this a joke?
I'm not from USA so I don't catch it
Is this a solidity/standards repo or a political one?

**Comment by fulldecent** [comment]
This is not a joke.

This is an Ethereum standards repo. This includes specifications which are designed to be read by humans and inspire participation and action so the word choice that affects human dignity is in-scope for this project.

A definition of politics is anything involving three or more people. This repo certainly includes more than three people therefore yes this is a political repo.

**Comment by MicahZoltu** [comment]
Authors are free to word their EIPs however they like.  We (editors) don't try to control what language authors choose to use or not use.  In this case, it was the author that wanted to make a non-normative change, which is allowed.

**Comment by Garito** [comment]
It is you believe that everything is politics and it is dumb as fu... that's my opinion
You will kick off a bunch of people from collaborating if this is about politics not about solidity
I can't take seriously this repo if you go this route since I give a fu... about this kind of religions

**Comment by fulldecent** [comment]
Your and my potential disagreement on the definition of the word "politics" is not constructively debated in this GitHub thread. Let us discuss on English Stack Exchange for any etymology or neologism, if you are interested. And there are other more serious places to discuss even than that.

I have authority (with coauthors) only over the publishing of _my_ EIPs. So please do not look down on other authors here based on interactions you and I have. The other authors can publish as they like—you can judge them on their own merits.

In other words, you may consider all the EIPs published here as op-eds without endorsement from the EIP Editors.

---

Meta: I do NOT think you are a troll. You have published contributions to web3 projects, [here's just one](https://github.com/Garito/svelte-web3/commit/4a5b5b6d1367eaeffcc775433feb741fb4d0da06) and are a prospective contributor to the EIPs repository.

**Comment by Garito** [comment]
You don't get it isn't it?
This is exactly why I think this pull request is a joke cause it isn't about solidity/standards but about USA politics and I, as many others, give a fu... about USA politics or politics in general
I'm asking to limit the theme of this repo to what it is SOLIDITY STANDARDS
You can go virtud signaling (or worst) to other places and not waste other people time with you religious believes
Focus, please, we have a lot of work to do to waste our time

**Comment by fulldecent** [comment]
There can be no constructive debate about religion. I'm here to build so I will not engage a discussion on that. 

Also, I will not take unpaid suggestions on how to use my time, you are welcome to lead by example.

As for the actual one-word change the instant subject of our discussion--I am happy to study more about how that word and other words affect people (in USA and elsewhere) but certainly not here.

---

P.S I have written a whole book mocking woke culture and neologisms. Thought you might be interested. It is called Nineteen Eighty-Five and it is a modernized copy of the Orwell book. Not available for sale in US or Spain.

**Comment by Garito** [comment]
YOU are wasting our time: "Remove racist language #4333" **Blacklist -> **Blocklist (How the fu... this has anything to do with SOLIDITY STANDARDS?)
I won't discuss any thing else here outside SOLIDITY STANDARDS anymore
You are not the center of the universe... If you don't see the issue it is your problem
Standards without people following them are no standards
